sonic-net/sonic-buildimage
# [Nokia7250][sonic-platform] Update sonic-platform submodule for Nokia-7250IXRE

c9f316e Disparate process and thread-safe protection for MDIPC transport, and refactored presence logic to better align with SfpStateUpdateTask operation
  a3486cc Added _get_module_bulk_info() and cache the info for 5 seconds to optimize the chassisd update.
  4b2e729 Fixed the nokia_cmd show qfpga help display
  7b87049 Fixed the nokia_cmd show midplane helper dispaly.
  83eabea Add "nokia_cmd set ndk-monitor-action" and "nokia_cmd set ndk-log-level" commands
  8aad7de Add nokia_cmd show ndk-version
  d2c55e3 Modify the psu.py and module.py to optimize the psud running time



#### Why I did it
Update sonic-platform submodule for Nokia-7250IXRE Platform

#### How I did it
Update submodule sonic-platform for Nokia-7250IXRE platform
   c9f316e Disparate process and thread-safe protection for MDIPC transport, and refactored presence logic to better align with SfpStateUpdateTask operation
    a3486cc Added _get_module_bulk_info() and cache the info for 5 seconds to optimize the chassisd update.
    4b2e729 Fixed the nokia_cmd show qfpga help display
    7b87049 Fixed the nokia_cmd show midplane helper dispaly.
    83eabea Add "nokia_cmd set ndk-monitor-action" and "nokia_cmd set ndk-log-level" commands
    8aad7de Add nokia_cmd show ndk-version
    d2c55e3 Modify the psu.py and module.py to optimize the psud running time
#### How to verify it
1) Execute following two show commands on Supervisor cards, and they should show the correct info
```
admin@ixre-cpm-chassis12:~$ show chassis modules status 
        Name             Description    Physical-Slot    Oper-Status    Admin-Status
------------  ----------------------  ---------------  -------------  --------------
FABRIC-CARD0                    SFM1                1         Online              up
FABRIC-CARD1                    SFM2                2         Online              up
FABRIC-CARD2                    SFM3                3         Online              up
FABRIC-CARD3                    SFM4                4         Online              up
FABRIC-CARD4                    SFM5                5         Online              up
FABRIC-CARD5                    SFM6                6         Online              up
FABRIC-CARD6                    SFM7                7         Online              up
FABRIC-CARD7                    SFM8                8         Online              up
  LINE-CARD0  Nokia-IXR7250E-36x400G                1         Online              up
  LINE-CARD1  Nokia-IXR7250E-36x400G                2         Online              up
  LINE-CARD2  Nokia-IXR7250E-36x400G                3         Online              up
  LINE-CARD3  Nokia-IXR7250E-36x400G                4         Online              up
  LINE-CARD4  Nokia-IXR7250E-36x400G                5         Online              up
  LINE-CARD5  Nokia-IXR7250E-36x400G                6         Online              up
  LINE-CARD6  Nokia-IXR7250E-36x400G                7         Online              up
  LINE-CARD7  Nokia-IXR7250E-36x400G                8         Online              up
 SUPERVISOR0   Nokia-IXR7250E-SUP-10                A         Online              up
admin@ixre-cpm-chassis12:~$ show chassis modules midplane-status 
      Name    IP-Address    Reachability
----------  ------------  --------------
LINE-CARD0    10.6.1.100            True
LINE-CARD1    10.6.2.100            True
LINE-CARD2    10.6.3.100            True
LINE-CARD3    10.6.4.100            True
LINE-CARD4    10.6.5.100            True
LINE-CARD5    10.6.6.100            True
LINE-CARD6    10.6.7.100            True
LINE-CARD7    10.6.8.100            True
admin@ixre-cpm-chassis12:~$ 
```
2) Execute these commands on the line cards
```
admin@sonic:~$ show chassis modules midplane-status 
       Name    IP-Address    Reachability
-----------  ------------  --------------
SUPERVISOR0    10.6.0.100            True
admin@sonic:~$ 

```
#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
